src/geo/leaflet/leaflet-cartodb-webgl-layer-group-view.js

```diff
@@ -0,0 +1,49 @@
+var TC = require('tangram.cartodb');
+var LeafletLayerView = require('./leaflet-layer-view');
+
+var LeafletCartoDBVectorLayerGroupView = window.L.TileLayer.extend({
```

Comment:
We can `var L = require('leaflet');` instead of relying on `window.L`.

There's already a base class for tiled layers https://github.com/CartoDB/cartodb.js/blob/v4/src/geo/leaflet/leaflet-tiled-layer-view.js. Not sure if inheriting from it will add any value, but just so that you're aware.

Should this even be a tiled layer? Not sure, but perhaps it could be a plain [L.Class](http://leafletjs.com/reference.html#class)?